research-institute-for-nature-and-forest.csl

```diff
@@ -7,63 +7,95 @@
     <link href="http://www.zotero.org/styles/research-institute-for-nature-and-forest" rel="self"/>
```

Comment:
One more question -- is there documentation for the style you could link to?

Reply:
We have an example report for the styling. A documentation URL has been added in 8a32152ea.